service/lambdaFunction/index.js

```diff
@@ -27,20 +28,21 @@ const regularRun = async () => {
 
     try {
         browser = await playwright.launchChromium();
-        const size = { width: 1280, height: 600 };
 
         context = await browser.newContext({});
         await context.tracing.start({ screenshots: false, snapshots: false });
 
         page = await context.newPage();
         page.on('load', async (data) => {
             visitedUrls.push(data.url());
-            // await pageHandler(data, count);
         });
-        capture = await saveVideo(
-            page,
-            path.join(__dirname, '..', '..', 'tmp', 'video.mp4'),
-        );
+        if (process.env.IS_RECORD === 'to_record') {
```

Comment:
If the value of `process.env.IS_RECORD` is something that the user inserts, the best thing to do when comparing it to a string is to first have it all in lowercase (something like if (process.env.IS_RECORD.toLowerCase() === 'to_record').